modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -975,68 +977,114 @@ public abstract class Base${schemaName}ResourceImpl
 		}
 	}
 
-	private Collection<Permission> _getPermissions(long companyId, List<ResourceAction> resourceActions, long resourceId, String resourceName, String[] roleNames) throws Exception {
-		_checkResources(companyId, resourceId, resourceName);
+	private Collection<Permission> _getPermissions(
```

Comment:
Can we autogenerate tests for this to ensure the fix works?

Reply:
Make sure you autogenerate a test that fails without this fix. In a separate PR.